### PR TITLE
$childIndex可能超出int上限

### PR DESCRIPTION
**Pull request by NanQi**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "minter/minter-php-bip-44",
```
Comment (grkamil) [inline composer.json:2]: You can use your fork. Do not replace the name of the project in the composer.

**Review by grkamil — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (NanQi) [reply composer.json:2]: ok

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nanqi/php-bip-44",
+  "name": "nanqi/minter-php-bip-44",
```
Comment (grkamil) [inline composer.json:2]: It should be: minter/minter-php-bip-44

**Review by grkamil — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (NanQi) [reply composer.json:2]: Do you want to publish composer？

**Pull request closed without merging**